src/libstd/collections/mod.rs

```diff
@@ -420,7 +420,8 @@
 #![stable(feature = "rust1", since = "1.0.0")]
 
 #[stable(feature = "rust1", since = "1.0.0")]
-pub use alloc::Bound;
+#[rustc_deprecated(reason = "moved to `std::ops::Bound`", since = "1.26.0")]
```

Comment:
We may want to also tag this with `#[doc(hidden)]` to prevent it showing up in the docs

Reply:
Sounds good, done.